test/nonregression/test_run_iotools.py

```diff
@@ -0,0 +1,241 @@
+# coding: utf8
```

Comment:
If I understand correctly, these set of tests did not exist before this PR, correct?

If so, it would be worth submitting a different PR for those not related to OASIS3. This PR should be specifically about adding OASIS3 conversion.

Reply:
It's a great initiative nonetheless, and it deserves its own separate review process imo.

Reply:
Probably a quick merge problem. An old file that I reworked. It probably has been converted to Jenkins since then. I ll remove it.